xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/VideoFilterShader.h

```diff
@@ -74,7 +74,7 @@ namespace Shaders {
     GLuint m_kernelTex1;
 
     // shader handles to kernel textures
-    GLint m_hKernTex;
+    GLint m_hKernTex = 0;
```

Comment:
https://github.com/xbmc/xbmc/blob/8405bbf242a73e7b88641c7db4a9808fd46a2f47/xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/VideoFilterShader.cpp#L163

this is something completely different to a vriable, m_hKernTex, that is used for in glGetUniformLocation